modules/theme/src/themes/default/assets/images/identity-providers/hypr.svg

```diff
@@ -0,0 +1,18 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Shall we add the license header to this SVG file as well?

Ref: https://github.com/wso2/identity-apps/blob/master/modules/theme/src/themes/default/assets/images/identity-providers/expert.svg?short_path=c0d4482#L1-L17

Reply:
addressed with [59287e2](https://github.com/wso2/identity-apps/pull/3790/commits/59287e24afb1a5a8a9d6a9eb5e09df6999b3b7e0)